packages/ui/src/components/transactions/TransactionItem.tsx

```diff
@@ -595,6 +597,8 @@ const TransactionItem: React.FC<{
 
                         {status === TransactionStatus.SUBMITTED &&
                             metaType === MetaType.REGULAR &&
+                            transactionCategory !==
```

Comment:
this conditional is getting a bit complex, shouldn't we move it to a variable that "explains" it?

Reply:
Done and removed some tornado logic